src/app/components/SubjectHeading/SubjectHeadingShow.jsx

```diff
@@ -135,53 +137,55 @@ class SubjectHeadingShow extends React.Component {
           uuid={uuid}
           key={this.context.router.location.search}
         />
-      {contextHeadings ?
-        <div
-          className="nypl-column-half subjectHeadingInfoBox"
-          tabIndex='0'
-          aria-label='Neighboring Subject Headings'
+        {contextHeadings ?
+          <div
+            className="nypl-column-half subjectHeadingInfoBox"
+            tabIndex='0'
+            aria-label="Neighboring Subject Headings"
           >
-          <div className="backgroundContainer">
-            <h4>Neighboring Subject Headings</h4>
+            <div className="backgroundContainer">
+              <h4>Neighboring Subject Headings</h4>
+            </div>
+            <SubjectHeadingsTable
+              subjectHeadings={contextHeadings}
+              location={location}
+              showId={uuid}
+              keyId="context"
+              container="context"
+              seeMoreLinkUrl={linkUrl}
+              seeMoreText="See more in Subject Headings Index"
```

Comment:
Why pass a hard coded property? Is this sort of future proofing that we may want to make this dynamic? Seems like a lot of passing for one property that could be determined closer to where it actually need to be implemented.

Reply:
I see two possible alternative approaches here: one is the one that this PR implements. The other approach is to have a conditional in the `AdditionalSubjectHeadingsButton`. I think I prefer this because 1. having conditionals all over the code seems messy and 2. This way the text is set in the component that actually decides what the text should be. Otherwise the button needs to think about where it is being rendered, which seems less modular. But in the end I could go either way.

Not sure I understand the future proofing comment, could you clarify?

Reply:
Future proofing because we were talking about making the text dynamic. 

Right now, I still think this unnecessary and adds on to something we both agree is a headache, which is the number of props being passed. I think if the responsibility of determining the text can be done by the component itself based on the props that are already being passed to `SubjectHeading`, why not just do that? This approach makes it trickier to follow the logic and debug later on, in my opinion.